example.js

```diff
@@ -12,7 +12,6 @@ const {
 
 console.log(toUnicode( 'Koj' ))
 console.log(toAscii('ਖੋਜ'))
-console.log(firstLetters( 'hir hir hir gun gwvhu ]' ))
```

Comment:
Why remove this example?

Reply:
Support for ascii has been removed. This is to make it possible to support transliterations without additional parameters, and there isn’t need anymore to support ascii directly, since we have `toUnicode()`